pagopa/io-app
# feat: [PE-617] Change labels in CGN Partner detail screen

## Short description
Change labels in CGN Partner detail screen

## List of changes proposed in this pull request
- change labels in localization files

## How to test
Related figma https://www.figma.com/design/yOoBGDr1MJpa0LY5lDUFSd/IOapp_CartaGiovaniNazionale?node-id=6102-37089&t=gaDY2KPmmBJvThH2-4
Navigate to highlighted screens to check new copy.
- login into the app -> navigate to "portafoglio" screen clicking on the icon in the bottom menu -> navigate to "carta giovani nazionale" by clicking on the the card in the wallet -> navigate to available partners list screen by clicking on the button on bottom of the page -> click on any category -> click on any partner -> check copy in this screen

@mariateresaventura 

[PE-617]: https://pagopa.atlassian.net/browse/PE-617?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
